util/splitProcessing.ts

```diff
@@ -12,13 +13,52 @@ import {asArray, XYCoordinates}                        from '~/util/util';
 import {secondsToLivesplitFormat, stringTimeToSeconds} from '~/util/durations';
 import {xmlParser}                                     from '~/util/xml';
 import {extractPng}                                    from './pngExtractor';
+import store                                           from './store';
 
 export interface DetailedSegment extends Segment {
   Subsplits: DetailedSegment[],
   Index: number, // Index in the original raw SplitFile
   IsSubsplit: boolean
 }
 
+const moveTransferTime = (chosenTime: OptionalRealAndGameTime, transferTime: number) => {
+  const newRealTime = stringTimeToSeconds(chosenTime.RealTime || "0.0.0.0") + transferTime;
+  chosenTime.RealTime = secondsToLivesplitFormat(newRealTime);
+  
+  if (store.state.hasGameTime) {
+    const newGameTime = stringTimeToSeconds(chosenTime?.GameTime || "0.0.0.0") + transferTime;
+    chosenTime.GameTime = secondsToLivesplitFormat(newGameTime);
+  }
+}
+
+export const moveTime = (currentSplit: Segment, otherSplit: Segment, transferTime: number) => {
+  const currentSplitTimes = currentSplit.SegmentHistory?.Time;
+  const otherSplitTimes = otherSplit.SegmentHistory?.Time;
+  const currentSplitComparisons = currentSplit.SplitTimes.SplitTime;
+  const otherSplitComparisons = otherSplit.SplitTimes.SplitTime;
+
+  moveTransferTime(currentSplit.BestSegmentTime, -transferTime);
+  moveTransferTime(otherSplit.BestSegmentTime, transferTime);
+
+  currentSplitTimes?.forEach((attemptCurrentSplit) => {
```

Comment:
When there is only 1 instruction you can use the shorter syntax : 
```TS
currentSplitTimes?.forEach(attemptCurrentSplit => moveTransferTime(attemptCurrentSplit, -transferTime));
```
(Same for all forEach just after)